packages/react/src/components/accordion/accordion-item.test.tsx

```diff
@@ -0,0 +1,46 @@
+import { mountWithTheme } from '../../test-utils/renderer';
+import { AccordionItem, Accordion } from './index';
+
+describe('Accordion Component', () => {
+    it('Should, on load, rewrite props and add to Accordion children correctly', () => {
+        const wrapper = mountWithTheme(
+            <AccordionItem title='Accordion 1' id="test1">
+                <div>Accordion content 1</div>
+            </AccordionItem>,
+        );
+
+        const accordionItem = wrapper.find('.accordion-content').at(0);
+        const accordionButton = wrapper.find('.accordion-button').at(0);
```

Comment:
Use `data-testid` instead of classes to find elements in your tests.

Reply:
I will see how to use data-tesetid.... do we have a documentation for this of I will need to dig in the files to see how it is used?

Reply:
using the elements instead of data-testid or class